### PR TITLE
Register optimize commands

### PR DESCRIPTION
**Pull request by erikgaal**
[Newly introduced in Laravel 11.27](https://github.com/laravel/framework/pull/52928), packages can now register the commands that should run during the `optimize` phase of deployment.

This PR adds the recommended steps for optimizing whenever it's available.

### CODE REVIEW

File: src/BladeIconsServiceProvider.php [hunk 1]
```diff
@@ -90,6 +90,14 @@ private function bootCommands(): void
                 Console\CacheCommand::class,
                 Console\ClearCommand::class,
             ]);
+
+            if (method_exists($this, 'optimizes')) {
+                $this->optimizes(
+                    optimize: 'icons:cache',
+                    clear: 'icons:clear',
+                    key: 'blade-icons'
+                );
```
Comment (driesvints) [inline src/BladeIconsServiceProvider.php:99]: Can we not use named arguments so we don't have to drop older PHP versions?

**Review by driesvints — CHANGES_REQUESTED** [review]
Thanks! Just one remark.

**Review by erikgaal — COMMENTED** [review]
done!

**Comment by driesvints** [comment]
Thanks!

**Pull request merged**